adapter/noop2/noop2.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 Istio Authors.
```

Comment:
Regarding your other comment, the model we want is that every single handler results in a single instance of an adapter at runtime. So given a Handler in config, this defines the block of state we should pass to NewAdapter.